lib/active_encode/engine_adapters/ffmpeg_adapter.rb

```diff
@@ -119,8 +118,8 @@ def write_errors encode
       def build_input encode
         input = ActiveEncode::Input.new
         metadata = get_tech_metadata(working_path("input_metadata", encode.id))
-        input.assign_tech_metadata metadata
         input.url = metadata[:url]
+        input.assign_attributes(metadata.tap { |h| h.delete(:url) })
```

Comment:
I think the intention would be clearer here if you avoided `tap` and either delete the url entry in the previous line or use `reject`:
`metadata.reject { |k,v| k == :url }`